frontend-mini-challenge/components/LoggedInPage.tsx

```diff
@@ -131,8 +155,15 @@ Then when you make use of this page component:
     </Page>
 
 */
-export async function getServerSidePropsForLoggedInPage({ query, req }) {
-  const whoAmIResponse = await serverSideWhoAmI({ query, req });
+
+export async function getServerSidePropsForLoggedInPage({
+  query,
+  req,
+}: {
+  query: ParsedUrlQuery;
+  req: IncomingMessage;
```

Comment:
Please use exactly this for req's type. There is no need to be creative here, we can just copy. 

Is there a reason you want to leave out the part about the cookies?

```
req: IncomingMessage & {
    cookies: NextApiRequestCookies
  }
```